beaker-vue/src/components/panels/FilePanel.vue

```diff
@@ -56,7 +56,16 @@
         <Column field="name" header="Name" class="filename-column" sortable>
           <template #body="slotProps">
             <span :class="[...getFileIconClass(slotProps.data), 'file-icon', slotProps.data.type]"></span>
-            <span :id="`file-list::${slotProps.data.path}`" class="file-name" :class="slotProps.data.type">
+            <span 
+              :id="`file-list::${slotProps.data.path}`" 
+              class="file-name" 
+              :class="slotProps.data.type"
+              @click="() => {
+                if (slotProps.data.type !== 'directory') {
+                  previewFile(slotProps.data.path, slotProps.data.mimetype)
```

Comment:
As we already have the filesize for each file, do we want to add a confirmation when the files are "very large" since they might have to be transferred over the network? Something like: `if filesize > 10mb: confirm("File ${filename} is ${filesize} big. Do you want to continue previewing?`